.github/workflows/build.yml

```diff
@@ -287,6 +287,18 @@ jobs:
           if ( ls ${ORION_PREFIX}/orion-dashboard/build/static/js/main.*.js ); then true; else false; fi
           echo Check if frontend script can find dashboard build
           python -c "from orion.core.cli.frontend import get_dashboard_build_path; get_dashboard_build_path();"
+
+          echo Check if frontend script can find dashboard build on sys.prefix/local
```

Comment:
Did you verify that this fails as expected without the fix?

Reply:
Hello @bouthilx ! Yes, I can confirm it fails as expected when using current PYPI package (`pip install orion`). But I don't know how to put it in CI tests.

Reply:
It's good enough to have it in the build here.